src/components/buttons/z-storia-button/index.stories.mdx

```diff
@@ -0,0 +1,19 @@
+import { html } from "lit-html";
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
+
+<Meta title="Buttons/ZStoriaButton" component="z-storia-button" />
```

Comment:
May I suggest to insert the story in the `Snowflakes/Storia` paths instead of official `Buttons` section?

Reply:
Sounds reasonable.